src/java.base/share/classes/sun/security/x509/EDIPartyName.java

```diff
@@ -61,7 +63,7 @@ public class EDIPartyName implements GeneralNameInterface {
      */
     public EDIPartyName(String assignerName, String partyName) {
         this.assigner = assignerName;
-        this.party = partyName;
+        this.party = Objects.requireNonNull(partyName);
```

Comment:
It looks like this constructor is not used.  Is if safe to remove it?

Reply:
Unused code means useless cost.  For private package, it may be nice to keep compact and no unused code.  As the class is public, I'm fine to leave as it is if you want to.